src/includes/tip-in-list.njk

```diff
@@ -1,7 +1,7 @@
 {% set nbSupportedBrowsers %}{{ post.data.tags | onlyBrowsers | length }}{% endset %}
 <li class="hover-box tip {{ "unique-browser-feature" if nbSupportedBrowsers == 1 }}">
   <a href="{{ post.url }}" class="tip-title">{{ post.data.title }}</a>
-  {% mainImage post %}
+  <a href="{{ post.url }}" class="tip-image">{% mainImage post %}</a>
```

Comment:
There are cases where `mainImage` returns null, and in this case there isn't an image to be displayed. When this happens, we shouldn't add the link either. 
I'll do this later, don't worry about it for now. This is a minor detail because there are only a few tips that don't have images.